recipe/build.sh

```diff
@@ -1,8 +1,6 @@
 #! /bin/bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
```

Comment:
> The copy fails because the files are read-only in the unpacked tarball.
> That's trivially worked around but in this package we regenerate them
> anyway, so there's no point to the copy.

Do you know if `$BUILD_PREFIX/share/gnuconfig/config.*` and the generated are equivalent in the macOS cross-compile setup?
Just asking out of curiosity (Isuru is the expert for this I know about; but I wouldn't want to bug him, if not necessary -- and maybe you have that expert knowledge too :).)

Reply:
I'm certainly not an expert but I'm 99% sure that in other X.org packages I've seen Isuru simply remove the copy command. You'll see that farther down in the `build.sh` the Autotools are rerun, which should hopefully have the same effect as the manual copy, and I think is why the copy can be skipped.

Reply:
> I'm 99% sure

Perfect! I was like 75 % certain, so that puts our (independent) certainties to a combined \~ 99.75 % -- good enough for me ^^.
Thanks!